python/lsst/afw/geom/transform/transform.cc

```diff
@@ -99,9 +99,9 @@ void declareTransform(py::module &mod, std::string const &fromName, std::string
      * of length 1 from being deleted */
     cls.def("_getJacobian", &Class::getJacobian);
 
-    declareMethodTemplates<GenericEndpoint, FromEndpoint, ToEndpoint>(cls);
-    declareMethodTemplates<Point2Endpoint, FromEndpoint, ToEndpoint>(cls);
-    declareMethodTemplates<SpherePointEndpoint, FromEndpoint, ToEndpoint>(cls);
+    declareMethodTemplates<FromEndpoint, ToEndpoint, GenericEndpoint>(cls);
```

Comment:
Yes, I thought about that. I'm still happier with it because I can instantly see that it matches the header file and other code.